octopus-server/src/main/java/octopus/teamcity/server/generic/PushPackageSubStepType.java

```diff
@@ -0,0 +1,54 @@
+package octopus.teamcity.server.generic;
+
+import java.util.List;
+import java.util.Map;
+
+import jetbrains.buildServer.serverSide.InvalidProperty;
+import octopus.teamcity.common.OverwriteMode;
+import octopus.teamcity.common.pushpackage.PushPackagePropertyNames;
+import org.apache.commons.compress.utils.Lists;
+
+public class PushPackageSubStepType extends OctopusBuildStep {
```

Comment:
Why is this a "sub" step, but the "others" aren't?

Reply:
good point! Stolen from a branch before the renames - resolved now.